_security-analytics/usage/overview.md

```diff
@@ -18,27 +18,31 @@ Each section provides a summary description for each element of Security Analyti
 
 ## Overview and getting started
 
-The upper-right corner of the Overview page contains two control buttons for refreshing information and getting started with Security Analytics. You can select the **Refresh** button to refresh all of the information on the page. You can also select the **Getting started** link to expand the Get started with Security Analytics window, which inludes a summary of the steps for set up as well as control buttons that allow you to jump to any of the steps.
-<img src="{{site.url}}{{site.baseurl}}/images/Security/get-started.png" alt="The getting started quick launch window">
+The upper portion of the Overview page contains two control buttons for refreshing information and getting started with Security Analytics. You can select the **Refresh** button to refresh all of the information on the page. 
+
+You can also select the **Getting started** link to expand the Get started with Security Analytics window, which inludes a summary of the steps for set up as well as control buttons that allow you to jump to any of the steps.
```

Comment:
```suggestion
You can also select the **Getting started** link to expand the Get started with Security Analytics window, which includes a summary of the steps for set up as well as control buttons that allow you to jump to any of the steps.
```

Reply:
Just a suggestion for shortening  >>> Select the **Getting started** link to expand the **Get started with Security Analytics** window, which includes a summary of the setup steps and control buttons that allow you to jump to any step.

Reply:
Better yet, >>The upper portion of the Overview page contains two options: Refresh and Getting Started with Security Analytics. Select **Refresh** to refresh the information on the page. Select **Getting Started** to display the Getting Started with Security Analytics window.  The Getting started for Security Analytics window includes a summary of the setup steps and options that allow you to jump to any step. (Again, just a suggestion, but could use a little tightening.)